src/components/quiz/UserDetailsForm.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+
+interface User {
+  name: string | null;
+  displayPic: string | null;
```

Comment:
There's some nice syntax for this that automatically allows `undefined` as a possible type:
```suggestion
  name?: string;
  displayPic?: string;
```

Reply:
Or you could actually not have these nullable at all. See below!